nima/websocket/webserver/src/main/java/module-info.java

```diff
@@ -14,10 +14,20 @@
  * limitations under the License.
  */
 
+import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.HelidonFlavor;
+
 /**
  * WebSocket server support.
  */
+@Feature(value = "WebsocketServer",
```

Comment:
Name is what is seen in the initial output. In this case it should be `WebSocket`.

Reply:
Fixed!